### ISSUE TITLE
Can't use in other modules/constructs

### ISSUE THREAD

**Issue by mbonig** [udondan/iam-floyd#27]
I tried incorporating cdk-iam-floyd into nightynight (https://github.com/mbonig/nightynight) and wakeywakey (https://github.com/mbonig/wakeywakey) but I'm getting a jsii build error:

```

> @matthewbonig/wakeywakey@0.1.0 tsc:build /home/matt/projects/cdk-sandbox/wakeywakey
> tsc

node_modules/cdk-iam-floyd/lib/shared/policy-statement/2-conditions.d.ts:6:20 - error TS2304: Cannot find name 'Condition'.

6     [key: string]: Condition;
                     ~~~~~~~~~


Found 1 error.

```

**Comment by udondan** [comment]
Interesting. When I prepared v0.52.0 I already noticed the `Condition` was the only remaining `@internal` declaration. I used to have a lot more but JSII did not like them. This was the only internal export JSII did not complain about. OK, no internals then. 😸

**Comment by udondan** [comment]
Looking at the code, it totally makes sense. 

```typescript
/**
 * A Condition
 *
 * @internal
 */
export interface Condition {
  [key: string]: String;
}

/**
 * A collection of Condition's
 */
export interface Conditions {
  [key: string]: Condition;
}
```

Of course when `Conditions` is not internal, neither can be `Condition`.

**Comment by udondan** [comment]
This should be fixed in [v0.54.1](https://github.com/udondan/iam-floyd/releases/tag/v0.54.1).

Please be aware, I just discovered other problems in the latest release(s) though, which I have [no immediate solution](https://github.com/aws/jsii/issues/1935#issuecomment-684637465) for. The order of method calls is important and some combinations currently are just not possible.

I checked your nightynight statement and this should be fine. But if you experience issues of methods being unavailable, it stems from this problem.

BTW, your nightynight statement can be simplified 😸 No need to generate an ARN, you can just pass the intanceID like so:

```typescript
new statement.Ec2().allow().toStopInstances().onInstance(props.instanceId)
```
